index.js

```diff
@@ -193,7 +196,11 @@ async function createCodeDeployDeployment(codedeploy, clusterName, service, task
         sha256: appSpecHash
       }
     }
-  }).promise();
+  };
+  if (codeDeployDescription) {
```

Comment:
If we have default value available I don't think we need to check if codeDeployDescription is empty or not. We can directly assign codeDeployDescription to description parameter inside  `await codedeploy.createDeployment`

Reply:
Looking at the code again, I'm doing what @allisaurus is suggesting which is to not pass the description parameter to the api request at all if it's not set.